yt_dlp/extractor/iwara.py

```diff
@@ -63,20 +65,27 @@ def _extract_formats(self, video_id, fileurl):
         # https://github.com/yt-dlp/yt-dlp/issues/6549#issuecomment-1473771047
         x_version = hashlib.sha1('_'.join((paths[-1], q['expires'][0], '5nFp9kmbNnHdAFhaqMvt')).encode()).hexdigest()
 
+        preference = qualities([
+            'preview', '360', '540', 'Source'
+        ])
```

Comment:
```suggestion
        preference = qualities(['preview', '360', '540', 'Source'])
```

Are you sure these are the only options?